src/components/autocomplete/js/autocompleteController.js

```diff
@@ -639,7 +679,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   /**
    * Clears the searchText value and selected item.
    */
-  function clearValue ($event) {
+  function clearValue (shouldClearText) {
```

Comment:
I made a change that seems to not break anything. I separated `clearSearchText` and `clearSelectedItem`. This allows me to call clearSearchText selectively.